### ISSUE TITLE
Cannot view the history (git log) of other git repos with the buttons in the Source Control side bar

### ISSUE THREAD

**Issue by tddschn** [DonJayamanne/gitHistoryVSCode#576]
I'm in a multi-root workspace with more than 1 git repos.

This is the Output when I clicked on the button, where `repo-A` is the git repo I viewed history with this extension for the first time, and `repo-B` is the repo I was trying to view the history of:
```
git remote show origin -n  (completed in 0.029s)
git log --full-history --pretty=oneline --date-order --decorate=full --skip=0 --max-count=100 master --follow -- <path-to-repo-B>  (completed in 0.038s)
fatal: <path-to-repo-B>: '<path-to-repo-B>' is outside repository at '<path-to-repo-A>'

git rev-list --full-history --count master -- <path-to-repo-B>  (completed in 0.047s)
fatal: <path-to-repo-B>: '<path-to-repo-B>' is outside repository at '<path-to-repo-A>'

git shortlog -e -s -n HEAD  (completed in 0.029s)

```

How do I make it work? Thanks!

**Comment by ole1986** [comment]
Are there special chars or spaces in the path?

**Comment by tddschn** [comment]
Nope. I'm on macOS 10.15.6, and both paths are made up of these chars only: `[0-9a-zA-Z/.-]`.

**Comment by ole1986** [comment]
This might also be resolved with PR #586

**Comment by ole1986** [comment]
Please reopen if v0.6.11 does not fix this issue
